client/src/header.jsx

```diff
@@ -9,34 +10,69 @@ export class Header extends React.Component {
 		super(props)
 		this.showNewSighting = this.showNewSighting.bind(this);
 		this.showAnimalList = this.showAnimalList.bind(this);
+		this.showLogin = this.showLogin.bind(this);
 		this.goHome = this.goHome.bind(this);
+		this.logUserOut = this.logUserOut.bind(this);
+		this.showLogout = this.showLogout.bind(this);
 	};
 
 	goHome() {
 		this.props.viewHomepage();
-	}
+	};
 
 	showNewSighting() {
 		this.props.addSighting();
 	};
 
 	showAnimalList() {
 		this.props.viewAnimals();
-	}
+	};
+
+	showLogin() {
+		this.props.viewLogin();
+	};
+
+	showLogout() {
```

Comment:
This pair of naming is confusing

Reply:
Would it be possible to do:
`showLogin() {
    this.props.showLogin();
}`

I've been keeping the function names separate, but it gets a little messy.

Reply:
https://github.com/DukeBaird/wildlife-tracker/pull/11/files#r691668334

I think we can ditch this way and clean it up

Reply:
Done in header.jsx and newuser.jsx